contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -34,8 +55,12 @@ def prepare(cls, options, round_manager):
   @classmethod
   def register_options(cls, register):
     register('--mypy-version', default='0.710', help='The version of mypy to use.')
-    register('--config-file', default=None,
+    register('--config-file', default='build-support/mypy/mypy.ini',
              help='Path mypy configuration file, relative to buildroot.')
+    register('--whitelist-tag-name', default='type_checked',
```

Comment:
Did you mean for the default to be None? I think you had this at one point.

I believe it should be, so that the default is to have no filtering (our prior behavior).